.eslintrc.js

```diff
@@ -49,6 +49,8 @@ module.exports = {
     'arrow-parens': ['warn', 'as-needed'],
     'arrow-spacing': 'warn',
     'no-duplicate-imports': 'warn',
+    // TODO: Set 'variables' to 'true' after fixing the existing issues
+    'no-use-before-define': ['warn', { 'functions': true, 'classes': true, 'variables': false }],
```

Comment:
Setting `variables` to `true` results in ~90 warnings so couldn't do that at this time.